client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -58,7 +54,7 @@ const Header: FunctionComponent< Props > = ( {
 				// Avoid `only_wordpressdotcom` — it seems to fail to find results sometimes
 				include_wordpressdotcom: true,
 				quantity: 1,
-				...( isFilledFormValue( siteVertical ) && { vertical: siteVertical.id } ),
+				...{ vertical: siteVertical?.id },
```

Comment:
I was hoping that `null`ish values would get dropped by `addQueryArgs` automatically, and/or ignored by the endpoint, but maybe that's naive :sweat_smile:

Reply:
I think it might be stripped. Try the following:

- At http://calypso.localhost:3000/gutenboarding
- In the browser console, `localStorage.removeItem('WP_DATA')` to clear persisted data, and reload
- Open the Network tab
- Pick a site vertical
- Enter a site title
- Replace the site vertical by a string for which no vertical exists, e.g. `foo`

Compare the corresponding network request to the previous ones:

![image](https://user-images.githubusercontent.com/96308/70047768-beb76d00-15d9-11ea-8b8c-704b19d0280f.png)

No `vertical`!

(Previous query, with a valid vertical specified:)

![image](https://user-images.githubusercontent.com/96308/70047824-e27ab300-15d9-11ea-9f3d-8b563448546d.png)